RELEASE-NOTES.md

```diff
@@ -1,42 +1,39 @@
 ## RELEASE NOTES
 
-### Version 4.0.0
```

Comment:
It kinda looks like you've ditched everything to do with `Version 4.0.0`, which seems like a bad idea.